dovecot/install.sh

```diff
@@ -100,6 +100,9 @@ protocol lda {
 } 	
 " >> /etc/dovecot/dovecot.conf
 
+# Kill all processes (Apache) listening on port 80 because this may prevent the start of NGINX
+fuser -k 80/tcp
```

Comment:
In that case, we can break something working on the server. I think we should stop installation if there port 80 is busy, not we should kill process which are not related with Easymail. What do you think about it?